src/SignalR/server/SignalR/test/Startup.cs

```diff
@@ -25,11 +22,16 @@ public void ConfigureServices(IServiceCollection services)
                 options.DefaultAuthenticateScheme = CookieAuthenticationDefaults.AuthenticationScheme;
                 options.DefaultChallengeScheme = CookieAuthenticationDefaults.AuthenticationScheme;
             }).AddCookie();
+
+            services.AddAuthorizationPolicyEvaluator();
```

Comment:
It's super wierd to me that this is separate from just authorization/authentication. It's a super undiscoverable name.

Reply:
See https://github.com/aspnet/AspNetCore/issues/6899

Reply:
https://github.com/aspnet/AspNetCore/issues/9208 to track this